docs/content/any/guides/policies/index.md

```diff
@@ -144,7 +145,7 @@ let's take a moment to put them in context. Rules are loaded into
 a **knowledge base**, a kind of specialized in-memory database that
 supports **queries** by pattern matching and logical inference.
 
-![Oso Architecture](getting-started/policies/arch.svg)
+![Oso Architecture](guides/policies/arch.svg)
```

Comment:
is there any difference between linking to `guides/policies/arch.svg` vs. `/guides/rbac`? (note the leading slash)